docs/BLTouch.md

```diff
@@ -1,50 +1,129 @@
-documentation or of your MCU. Using the original wiring, the yellow wire from the tripple is
+documentation or of your MCU. Using the original wiring, the yellow wire from the triple is
 the `control_pin`, the white wire from the pair is the `sensor_pin`. You need to configure
 these pins according to your wiring with Klipper:
 
+```
+
     [bltouch]
     sensor_pin: P1.24
     #   Pin connected to the BL-Touch sensor pin. This parameter must be
     #   provided.
     control_pin: P1.26
+```
 
-Calibrating the BL-Touch
-========================
+You also most likely need to configure a homing override to first raise the z-axis, then home the x-
+and y-axis, and finally move to the center and home the z-axis. Like this:
 
-### Z Offset
-In order to make klipper work properly you need to tell it in which relation to the nozzle
-the probe is exactly located. Lets start with the `z_offset` - the height of the probe when
-its expanded:
+```
+    [homing_override]
+    gcode:
+        G1 Z10 ; Move up 10mm
+        G28 X Y
+        G1 X166 Y120 F6000 ; Change the X and Y coordinates to the center of your print bed
+        G28 Z
+    set_position_z: 0.0
+```
+
+It's important that the upwards movement is high enough, so that the probe or the nozzle don't hit
+any binder clips, even if the probe pin happens to be in it's lowest state.
 
-Make sure the probe is above the bed by moving the head into an appropriate x/y position:
+Initial tests
+=============
 
-    g0 x0 y0
+Before moving on, verify that the BLTouch is mounted at the correct height, the pin should be
+roughly 2 mm above the nozzle when retracted
 
-Now make klipper aproach the bed until the probe triggers using:
+When you turn on the printer, the BLTouch probe should perform a self-test and move the pin up and
+down a couple of times. Once the self-test is completed, the pin should be retracted and the red LED
+on the probe should be lit. If there are any errors, for example the probe is flashing red or the
+pin is down instead of up, please turn off the printer and check the wiring and configuration.
 
-    PROBE
+If the above is looking good, it's time to test that the probe responds to commands from the
+firmware. First run `BLTOUCH_DEBUG COMMAND=pin_down` in your printer terminal. Verify that the pin
+moves down, and that the red LED on the probe turns off. If not, check your wiring and configuration
+again. Next issue a `BLTOUCH_DEBUG COMMAND=pin_up` and verify that the pin moves up, and that the
+red light turns on again. If it's flashing then there's some problem.
 
-This should deploy the BL-Touch probe, stop the run when it triggers by touching the bed.
-Now acquire the current position from the reached `z_offset` using:
+Now, it's time to test homing with a twist. Instead of letting the probe pin touch the print bed,
+let it touch the nail on your finger. So issue a `G28`, wait until it starts to move down, and stop
```

Comment:
on a delta g28 moves up ;-) maybe @KevinOConnor knows better what to write here?

Reply:
I can change it to "wait until the z-axis homing movement starts".

Reply:
most probably `PROBE` is the thing you want to do on both style of printers - with the power knob in the hand. But kevin can tell best here.

Reply:
I don't think `PROBE` will work without first issuing a `G28`, since it returns a result relative to the homing position. But I haven't tested it, so I might be wrong. 

But otherwise, I guess you are right, moving just the z-axis and nothing else, would generate less confusion.